unstructured/ingest/runner/__init__.py

```diff
@@ -24,6 +26,35 @@
 from .slack import slack
 from .wikipedia import wikipedia
 
+runner_map: t.Dict[str, t.Callable] = {
```

Comment:
nothing actionable here, but realizing we've got a lot of these lists that we now have to maintain. also not sure that we've got better alternatives, just thinking outloud.